import_or_skip.sh

```diff
@@ -0,0 +1,8 @@
+# if there are more than 0 lines with importorskip, go into the if branch
```

Comment:
Will prevent us from using importorskip in the future:

![image](https://user-images.githubusercontent.com/41651716/144921839-de59a727-608a-4dbe-bd8d-89791938a04f.png)

Reply:
Really cool idea, thank you for adding this as well as fixing all of the current references to `importorskip`! 😁